### ISSUE TITLE
import bug fix in the documentation 

### ISSUE THREAD

**Issue by 12rambau** [diku-dk/bfast#22]
I tried to use the bfast documentation and it seems that is it sno longer up to date. 
To crop the data you suggested to use : 
```python
from bfast.utils import crop_data_dates
start_hist = datetime(2002, 1, 1)
start_monitor = datetime(2010, 1, 1)
end_monitor = datetime(2018, 1, 1)
data, dates = crop_data_dates(data_orig, dates, start_hist, end_monitor)
print("First date: {}".format(dates[0]))
print("Last date: {}".format(dates[-1]))
print("Shape of data array: {}".format(data.shape))
```

But `utils` is not part of bfast anymore. I created a PR to correct the documentation, feel free to accept it or change the `__init__.py`of the bfast module to integrate `utils` in it.

**Comment by mortvest** [comment]
This has been fixed on develop branch. It is getting added to master soon

**Comment by mortvest** [comment]
fixed with 0.7
